diff --git a/external/Catch2 b/external/Catch2
index fee8162..b3fb4b9 160000
--- a/external/Catch2
+++ b/external/Catch2
@@ -1 +1 @@
-Subproject commit fee81626d2a4811095c3a39d20fb355eeb954101
+Subproject commit b3fb4b9feafcd8d91c5cb510a4775143fdbef02f
diff --git a/scripts/initialize_ebpf_repo.ps1 b/scripts/initialize_ebpf_repo.ps1
index ff6a22a..1f60471 100644
--- a/scripts/initialize_ebpf_repo.ps1
+++ b/scripts/initialize_ebpf_repo.ps1
@@ -102,7 +102,7 @@ $nugetRestoreCommand = if ($msbuildDir) {
 $commands = @(
     "git submodule update --init --recursive",
     "cmake $cmakeCommonArgs -S external\ebpf-verifier -B external\ebpf-verifier\build -DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded$<$<CONFIG:Debug>:Debug>$<$<CONFIG:FuzzerDebug>:Debug>",
-    "cmake $cmakeCommonArgs -S external\catch2 -B external\catch2\build -DBUILD_TESTING=OFF -DCATCH_CONFIG_EXPERIMENTAL_THREAD_SAFE_ASSERTIONS=ON -DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded$<$<CONFIG:Debug>:Debug>$<$<CONFIG:FuzzerDebug>:Debug>",
+    "cmake $cmakeCommonArgs -S external\catch2 -B external\catch2\build -DBUILD_TESTING=OFF -DCATCH_CONFIG_THREAD_SAFE_ASSERTIONS=ON -DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded$<$<CONFIG:Debug>:Debug>$<$<CONFIG:FuzzerDebug>:Debug>",
     "cmake $cmakeCommonArgs -S external\ubpf -B external\ubpf\build -DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded$<$<CONFIG:Debug>:Debug>$<$<CONFIG:FuzzerDebug>:Debug>",
     "cmake $cmakeCommonArgs -S external\ubpf -B external\ubpf\build_fuzzer -DUBPF_ENABLE_LIBFUZZER=on",
     $nugetRestoreCommand
